storagemarket/types.go

```diff
@@ -582,6 +588,8 @@ type StorageClient interface {
 	// GetAsk returns the current ask for a storage provider
 	GetAsk(ctx context.Context, info StorageProviderInfo) (*SignedStorageAsk, error)
 
+	GetDealStatus(ctx context.Context, info StorageProviderInfo, proposalCid cid.Cid) (*ProviderDealState, error)
```

Comment:
I feel somewhat strongly this method should be called GetProviderDealState or GetDealStatusFromProvider to disambiguate from GetLocalDeal

Reply:
Done